### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (span[data-v-3b24e28d=""])

### ISSUE THREAD

**Issue by ertascan** [ertascan/ats#2]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[İlanlar | Kariyer.net](https://ats.kariyer.net/ilanlarim/yayindakiler)
#### Element path
a[href$="pasifler"] > span[data-v-3b24e28d=""]
#### Snippet
`<span data-v-3b24e28d="">1173</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.61 (foreground color: #999999, background color: #f5f5f5, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 112.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
